gravitee-apim-console-webui/src/management/home/widgets/gio-quick-time-range/gio-quick-time-range.component.html

```diff
@@ -23,7 +23,7 @@
     </mat-option>
   </mat-select>
 
-  <button matSuffix mat-icon-button class="time-frame__refresh-btn" matTooltip="Refresh" (click)="onRefresh()">
+  <button matSuffix mat-icon-button class="time-frame__refresh-btn" matTooltip="Refresh" (click)="onRefresh($event)">
```

Comment:
chore: remove `$event`

Reply:
needed to not open "selection" on click to refresh button